app/controllers/pages_controller.rb

```diff
@@ -1,4 +1,38 @@
+# frozen_string_literal: true
+
 class PagesController < ApplicationController
-  def home
+  include HTTParty
+
+  CLOVERPOP_DOMAIN = ENV['CLOVERPOP_DOMAIN'].freeze
+  CLOVERPOP_ORG_API_TOKEN = ENV['CLOVERPOP_ORG_API_TOKEN'].freeze
+
+  def home; end
+
+  def init_data
+    api_url = "#{CLOVERPOP_DOMAIN}/api/v1/decisions"
```

Comment:
@SerhiiBorozenets I think we could mode `"#{CLOVERPOP_DOMAIN}/api/v1/decisions"` to the controller constant as well like a 

DECISIONS_`API_URL = "#{ENV['CLOVERPOP_DOMAIN']}/api/v1/decisions".freeze`